src/main/cpp/events/sampleevent.cpp

```diff
@@ -360,7 +360,7 @@ void SampleEvent::mixBuffer( AudioBuffer* outputBuffer, int bufferPosition,
                         _readPointer = 0;
 
                     // use range pointers to read within the specific buffer ranges
-                    for ( c = 0, ca = _buffer->amountOfChannels; c < ca; ++c )
+                    for ( c = 0, ca = outputChannels; c < ca; ++c )
                     {
```

Comment:
By using the outputChannels (together with _mixMono_) we sum mono samples across a multi channel output.